plugin/data_masking/src/udf/udf_gen_dictionary_drop.cc

```diff
@@ -101,8 +101,10 @@ char *gen_dictionary_drop(UDF_INIT *, UDF_ARGS *args, char *result,
   DBUG_ENTER("gen_dictionary_drop");
 
   std::string res = _gen_dictionary_drop(args->args[0]);
-  *length = res.size();
-  strcpy(result, res.c_str());
+
+  strncpy(result, res.c_str(), *length - 1);
```

Comment:
from what I see, *length is 766. Better to move the *length calculation statement before.
1. debug assert that (res.size() < *length)
2. *length = min(*length-1, res.size)
3. strncpy(result, res.c_str(), *length)
4. result[*length]='\0';